### ISSUE TITLE
Remove "cloud" references

### ISSUE THREAD

**Issue by ijstokes** [Anaconda-Platform/nb_anacondacloud#29]
This is a really valuable feature of Anaconda Enterprise and the extension works well in an enterprise (i.e. no "cloud" connection) environment.  But enterprise customers will be confused when they click on the "upload" link and it says the content will go to Anaconda Cloud (first line and last line both have references).

We need one of the following:
- two versions of this extension
- configurable
- no references to cloud

I think the last is easiest, but the second option is probably the best.

**Comment by bollwyvl** [comment]
Indeed! Combining this with #14 is probably a good idea. We'd refactor out the UI as a baseline, `nb_publish` with whatever the absolute minimal publishing option is... SFTP? HTTP Post? Anonymous, secret gist? The default viewing experience would then be nbviewer, which can consume just about anything.

We'd leave the rest of the github/bitbucket/stash/sharepoint uploading as an exercise for the community as separate packages.

Then, `nb_publish_XXX` would be the specific-bits, which would depend on `nb_publish`. In the case of `nb_publish_anaconda`, from the API root (which would be detected from the API base... or manually configured), e.g. `http://api.anaconda.org` we can pull the `main_url`, fetch that, and show the `title` from there, however it is configured in `BRAND`.

**Comment by timsnyder** [comment]
Does nb_anacondacloud's relevence in the Enterprise go away with Anaconda Enterprise 5.0?  If not, I might submit a PR for this after fixing it for myself.

cc: @koverholt

**Comment by koverholt** [comment]
Correct @timsnyder, we will no longer be using this specific extension in Anaconda Enterprise v5 to move notebooks around (e.g., from AEN to Repo or AEN to Cloud). Thus, any messaging/references related to uploading assets to Anaconda Enterprise v5 vs. Anaconda Cloud will be resolved in places other than this repo/extension.
